src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Sql/Sprocs/RebuildIndex_2.sql

```diff
@@ -0,0 +1,60 @@
+﻿/*************************************************************
+    Stored procedures for rebuild index
+**************************************************************/
+--
+-- STORED PROCEDURE
+--     RebuildIndex
+--
+-- DESCRIPTION
+--     Stored procedures for rebuild index
+--
+-- PARAMETERS
+--     @tableName
+--         * index table name
+--     @indexName
+--         * index name
+--     @pageCompression
+--         * index page compression
+
+GO
+CREATE PROCEDURE dbo.RebuildIndex_2
+    @tableName nvarchar(128),
+    @indexName nvarchar(128),
+    @pageCompression bit
+WITH EXECUTE AS 'dbo'
+AS
+DECLARE @errorTxt as varchar(1000)
+       ,@sql as nvarchar (1000)
+       ,@isDisabled as bit = 0
+       ,@isExecuted as int = 0
+
+IF object_id(@tableName) IS NULL
+BEGIN
+    SET @errorTxt = @tableName +' does not exist or you don''t have permissions.'
+    RAISERROR(@errorTxt, 18, 127)
+END
+
+SET @isDisabled = (SELECT is_disabled FROM sys.indexes WHERE object_id = object_id(@tableName) AND name = @indexName)
+IF @isDisabled IS NULL
+BEGIN
+    SET @errorTxt = @indexName +' does not exist or you don''t have permissions.'
+    RAISERROR(@errorTxt, 18, 127)
+END
+
+IF @isDisabled = 1
+BEGIN
+	IF @pageCompression = 0 
+	BEGIN
+		SET @sql = N'ALTER INDEX ' + QUOTENAME(@indexName) + N' on ' + @tableName + ' Rebuild'
+	END
+	ELSE 
+	BEGIN
+		SET @sql = N'ALTER INDEX ' + QUOTENAME(@indexName) + N' on ' + @tableName + ' Rebuild WITH (DATA_COMPRESSION = PAGE)'
+	END
+
+	EXECUTE sp_executesql @sql
+	SET @isExecuted = 1
+END
+        
+RETURN @isExecuted
+GO
```

Comment:
Please remove.

Reply:
Currently we do not use it in product code but in test we use this to make sure the if rebuild on the index not disabled should skip. And remove the flag for disable index.

Reply:
Hmm... 

So you don't trust stored procedure to skip index rebuild correctly but trust it to set flag on the same condition? 

This sounds incorrect.

Please proceed with removing this flag. You may need to remove related to this flag test as well.

If you need help in writing test that truly checks whether index was rebuild please let me know.

Reply:
Remove this flag, and later we can apply new rebuild index logic.